packages/graphql-codegen-typescript-typemap-plugin/src/index.ts

```diff
@@ -16,7 +16,7 @@ const graphqlCodegenTypeMapPlugin: PluginFunction = (
         (typeName) =>
           `  "${typeName}": ${
             isScalarType(typesMap[typeName])
-              ? `Scalars["${typeName}"]`
+              ? `Scalars["${typeName}"]["output"]`
```

Comment:
@pavelglac Would this not break the plugin with our codegen version in TMP?

Reply:
Yes, without also bumping @graphql-codegen/typescript that would be a problem. We just need to bump two packages and then we should be good (have quickly test it and seems there wont be any problem).